src/dms_sync/azext_dms/_help.py

```diff
@@ -32,16 +31,16 @@
 
 helps['dms project task create'] = """
     type: command
-    short-summary: Create a migration Project which can contain multiple Tasks.
+    short-summary: Create and start a migration Task.
```

Comment:
nit: Why do we need to capitalize 'Task'?

Reply:
It's already a convention we follow in the GA version of the DMS CLI tool.

Reply:
Got it.